### ISSUE TITLE
ykman fails to connect to YubiKey when OATH code is being generated with the yubico-authenticator

### ISSUE THREAD

**Issue by joshdk** [Yubico/yubikey-manager#28]
### Version information

`macOS Sierra 10.12.6`

```
$ ykman --version
YubiKey Manager (ykman) version: 0.4.5
Libraries:
    libykpers 1.18.0
    libu2f-host 1.1.4
    libusb 1.0.21

$ ykman info
Device name: YubiKey 4
Serial number: 4039699
Firmware version: 4.2.7
Enabled connection(s): OTP+U2F+CCID

Device capabilities:
    OTP:	Enabled
    U2F:	Enabled
    CCID:	Enabled
    OPGP:	Enabled
    PIV:	Enabled
    OATH:	Enabled
```

### Steps to reproduce

- In yubico-authenticator, right-click on one of the slots and select "Generate code".
- Verify that yubico-authenticator prompts "Touch your YubiKey to generate the code.".
- In a terminal, run `ykman oath code` (or `ykman oath list`).
- Verify that `ykman` is waiting. (doesn't return instantly)
- Touch your YubiKey in response to the original yubico-authenticator request.
- Verify that yubico-authenticator successfully generates an OATH code.
- Observe that `ykman` **fails** with the following:
```
$ ykman oath code
Usage: ykman [OPTIONS] COMMAND [ARGS]...

Error: Failed connecting to the YubiKey.
```

Interestingly, if you use `ykman` itself to generate the OATH codes, (by running `ykman oath code <slot name>`) both operations succeed.

**Comment by emlun** [comment]
Interesting. Thank you for the report! This will probably be low priority to fix, though, considering the rather contrived edge case where it happens.

What version of Yubico Authenticator did you use for this?

**Comment by joshdk** [comment]
Hi @emlun, thanks for taking a look anyway! I mainly wanted to file this in the event that it might be a symptom of a deeper problem. (like a lock not being release somewhere?)

I'm using version `4.2.0` of the Yubico Authenticator.

**Comment by emlun** [comment]
This seems to be fixed in ykman `0.4.6`. Thanks for reporting! Feel free to reopen the issue if you're still having this problem after upgrading.
